src/js/demo/app.jsx

```diff
@@ -1,21 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import moment from 'moment';
-import Calendar from 'calendarjs';
 
 import '../../styles/main.scss';
 
-import Month from './../components/month.jsx';
+import ReactTimeslotCalendar from './../react-timeslot-calendar.jsx';
+import { DEFAULT_TIMESLOTS } from './../constants/day';
 
 export default class App extends React.Component {
   render() {
-    let cal = new Calendar();
     return (
       <div>
-        Timeslot CSS Test (Updated App)
-        <Month
-          date = { moment() }
-          weeks = { cal.generate() }
+        React Timeslot Calendar!
+        <ReactTimeslotCalendar
+          initialDate = { moment().format() }
```

Comment:
`initialDate` and `currentDate` are way too different.

Initial Date -> The date I provided when the component was initialized.
Current Date -> After moving around weeks/months, currentDate is the date in which the user finds himself in 

Meaning, I can start with `initialDate = now()`, but if I want to schedule on Next Week, `currentDate` would be similar to `initialDate + 1week`.